peerconnection_media_test.go

```diff
@@ -1280,7 +1280,28 @@ func TestPeerConnection_Simulcast(t *testing.T) {
 
 		assert.NoError(t, signalPair(pcOffer, pcAnswer))
 
-		for sequenceNumber := uint16(0); !ridsFullfilled(); sequenceNumber++ {
+		// padding only packets should not affect simulcast probe
+		var sequenceNumber uint16
+		for sequenceNumber = 0; sequenceNumber < simulcastProbeCount+10; sequenceNumber++ {
+			time.Sleep(20 * time.Millisecond)
+
+			for _, track := range []*TrackLocalStaticRTP{vp8WriterA, vp8WriterB, vp8WriterC} {
+				pkt := &rtp.Packet{
+					Header: rtp.Header{
+						Version:        2,
+						SequenceNumber: sequenceNumber,
+						PayloadType:    96,
+						Padding:        true,
+					},
+					Payload: []byte{0x00, 0x02},
```

Comment:
@cnderrauber Sorry, missed this. This has two byte payload, but the check for padding only checks for `Padding == true` && payload length is zero. Here it is not. I am guessing this is not fulfilling rids because the extension is not set.

Reply:
Oops, please ignore me. The payload last byte is padding size. My bad. Sorry about the fake news.